Jenkinsfile

```diff
@@ -37,7 +37,7 @@ stage('prep') {
         dir('target') {
             plugins = readFile('plugins.txt').split('\n')
             lines = readFile('lines.txt').split('\n')
-            lines = [lines[0], lines[-1]] // run PCT only on newest and oldest lines, to save resources
+            lines = [lines[0]] // run PCT only on newest and oldest lines, to save resources
```

Comment:
Is the comment still valid?

```suggestion
            lines = [lines[0]] // run PCT only on newest line, to save resources
```

Reply:
This PR isn’t intended to be merged so comments don’t matter

Reply:
I see!

Reply:
Code reviews don’t matter, either